packages/zendesk-adapter/src/filters/article/utils.ts

```diff
@@ -199,3 +205,57 @@ export const createUnassociatedAttachment = async (
     throw getZendeskError(attachmentInstance.elemID.getFullName(), err)
   }
 }
+
+export const deleteArticleAttachment = async (
+  client: ZendeskClient,
+  attachmentInstance: InstanceElement,
+): Promise<void> => {
+  const res = await client.delete({
+    url: `/api/v2/help_center/articles/attachments/${attachmentInstance.value.id}`,
+  })
+  if (res === undefined) {
+    log.error('Received an empty response from Zendesk API when deletd an article attachment')
+  }
+}
+
+const getResolvedParent = (resolvedInstance: InstanceElement): Values | undefined => {
+  const instanceParentsList = resolvedInstance.annotations[CORE_ANNOTATIONS.PARENT]
+  if (!_.isArray(instanceParentsList)) {
+    return undefined
+  }
+  return instanceParentsList[0]
+}
```

Comment:
the existing functions support single values as well and not only arrays - please use 
```suggestion
const getResolvedParent = (resolvedInstance: InstanceElement): Values | undefined => (
  getParents(resolvedInstance)[0]
)
```
if you can't just use that function directly